src/abstractdataframe/abstractdataframe.jl

```diff
@@ -551,11 +551,13 @@ Indexes of complete cases (rows without NA's)
 
 ```julia
 nonunique(df::AbstractDataFrame)
+nonunique(df::AbstractDataFrame, columns)
 ```
 
 ### Arguments
 
 * `df` : the AbstractDataFrame
+* `columns` : specifies the columns to compare
```

Comment:
Since `columns` is also a function name, might want to rename this to `cols` (which, e.g., is the name used in `sort`).